src/plugins/vis_types/vislib/public/vislib/VISLIB.md

```diff
@@ -20,5 +20,4 @@ All base visualizations extend from `visualizations/_chart`
 
 `visualizations/point_series` takes care of drawing the point series chart (no axes or titles, just the chart itself). It creates all the series defined and calls render method on them.
 
-currently there are 3 series types available (line, area, bars), they all extend from `visualizations/point_series/_point_series`.
-
+Vislib supports the heatmap and gauge/goal charts from the aggregation-based visualizations. It also contains the legacy implemementation of the pie chart (enabled by the visualization:visualize:legacyPieChartsLibrary advanced setting).
```

Comment:
I think mentioning the Pie chart should be moved to line 15 of this file because it's not point series chart.

Reply:
Nit: Ok, looking at the other changes I think maybe we could rename the section `Point series chart` to `heatmap chart` (because it's the only case now) and specify the description.

Reply:
I will move it in the beginning with title `Vislib charts supported`